specification/monitor/resource-manager/readme.go.md

```diff
@@ -130,4 +136,58 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag) == 'package-2021-09' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2021-09-01-preview/$(namespace)
-```
\ No newline at end of file
+```
+
+### Tag: package-2022-03-preview-datacollection and go
+
+These settings apply only when `--tag=package-2022-03-preview-datacollection --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-preview-datacollection' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2022-03-26-preview/$(namespace)
+```
+
+### Tag: package-2022-03-part5 and go
+
+These settings apply only when `--tag=package-2022-03-part5 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-part5' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2022-03-25/$(namespace)
+```
+
+### Tag: package-2022-03-part4 and go
+
+These settings apply only when `--tag=package-2022-03-part4 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-part4' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2022-03-24/$(namespace)
+```
+
+### Tag: package-2022-03-part3 and go
+
+These settings apply only when `--tag=package-2022-03-part3 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-part3' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2022-03-23/$(namespace)
```

Comment:
can we put `Microsoft.Insights/stable/2015-04-01/activityLogs_API.json` in this tag as well?
Is activityLogs related with activityLogAlerts? They have similar names
```suggestion
output-folder: $(go-sdk-folder)/services/monitor/mgmt/2020-10-01/activityLogAlerts
```
If we could combine `activityLogs` in this tag, we could change the name to `activityLogs`